src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/ReferenceCodeServiceTest.kt

```diff
@@ -45,14 +45,42 @@ class ReferenceCodeServiceTest {
   }
 
   @Test
-  fun `Should return a list of of ordered references codes by display sequence for a given group code`() {
+  fun `Should return a list of of ordered references codes by display sequence`() {
     val one = referenceCodeEntity(1L, "GROUP_CODE", "SENTENCE", "Sentence hearing", displaySequence = 1)
     val two = referenceCodeEntity(3L, "GROUP_CODE", "COMMITTAL", "Committal", displaySequence = 2)
     val three = referenceCodeEntity(2L, "GROUP_CODE", "APPEAL", "Appeal", displaySequence = 3)
 
     whenever(referenceCodeRepository.findAllByGroupCodeEquals("GROUP_CODE")).thenReturn(listOf(two, three, one))
 
-    service.getReferenceDataByGroup("GROUP_CODE", false) containsExactly listOf(one, two, three).toModel()
+    service.getReferenceDataByGroup("GROUP_CODE", false) containsExactly listOf(one.toModel(), two.toModel(), three.toModel())
```

Comment:
This test was wrong, now fixed.  Missed the fact another sort was being done in the mapper object (which is shouldn't have been).  The service should be doing the sorting.